### DISCUSSION TITLE
[C#12] reduce namespace indentation

### DISCUSSION THREAD

**Discussion by emako** [Apollo3zehn/FluentModbus#131] [Ideas]
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/namespace

use `namespace FluentModbus;` to reduce indentation.

**Comment by emako** [comment]
@Apollo3zehn

**Comment by Apollo3zehn** [comment]
Please create a P/R, I would love to use the simplified namespace definition in this project.

**Comment by emako** [comment]
All right, create a PR later
